packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/storage/simcore_s3.py

```diff
@@ -0,0 +1,29 @@
+from models_library.api_schemas_rpc_async_jobs.async_jobs import (
+    AsyncJobGet,
+    AsyncJobNameData,
+)
+from models_library.api_schemas_storage import STORAGE_RPC_NAMESPACE
+from models_library.api_schemas_storage.storage_schemas import FoldersBody
+from models_library.rabbitmq_basic_types import RPCMethodName
+from models_library.users import UserID
+
+from ..._client_rpc import RabbitMQRPCClient
+from ..async_jobs.async_jobs import submit
```

Comment:
MINOR: this doubly nexted  `async_jobs` is strange 

What about

```python

from ..async_jobs import async_jobs

async def copy_folders_from_project(...)
      await async_jobs.submit(...)

```

Reply:
yep but is correct